tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -327,7 +329,7 @@ public BloomFilter readBloomFilter() throws IOException {
     return deviceMetadata;
   }
 
-  public TimeseriesMetadata readTimeseriesMetadata(Path path, boolean ignoreNotExists)
+  public ITimeSeriesMetadata readTimeseriesMetadata(Path path, boolean ignoreNotExists)
```

Comment:
this method is used by the server, where we know whether the timeseries are aligned or not. So no need to change this method you could write a new method (readITimeseriesMetadata) for TsFile API.